core/modules/simpletest/tests/database_test.test

```diff
@@ -3184,7 +3188,7 @@ class DatabaseInvalidDataTestCase extends DatabaseTestCase {
  */
 class DatabaseQueryTestCase extends DatabaseTestCase {
   /**
-   * Test that we can specify an array of values in the query by simply passing in an array.
+   * Test that we can specify an array of values in the query by passing in an array.
```

Comment:
```suggestion
   * Test that we can specify an array of values in the query, by passing in an
   * array.
```

Reply:
One could argue this is a "restrictive clause" so the comma should not be there. https://www.grammarly.com/blog/comma/